examples/example-nextjs/package.json

```diff
@@ -10,19 +10,20 @@
     "example:start": "next start"
   },
   "dependencies": {
-    "@stylexjs/stylex": "0.11.1",
     "@stylexjs/open-props": "0.11.1",
+    "@stylexjs/stylex": "0.11.1",
     "next": "14.2.21",
     "react": "^18.3.0",
-    "react-dom": "^18.3.0"
+    "react-dom": "^18.3.0",
+    "styled-jsx": "^5.1.1"
```

Comment:
Why do we need this dependency?

Reply:
This is to fix the build error in example-nextjs.

The root cause is that multiple versions of React are used in this repository. Specifically, React v17.0.2 is defined in docs, causing version 17 to be installed in the root’s node_modules. Meanwhile, examples/example-nextjs specifies React v18 directly in its own package.json, causing version 18 to be installed in its local node_modules.

Although example-nextjs typically uses React v18, the React invoked via styled-jsx during the build process was incorrectly referencing React v17. To address this, a temporary adjustment was made so that React called through styled-jsx would also reference examples/example-nextjs/node_modules.

Reply:
Strange, but makes sense